cmd/lakectl/cmd/commit.go

```diff
@@ -12,13 +14,15 @@ import (
 
 var errInvalidKeyValueFormat = errors.New(`invalid key/value pair - should be separated by "="`)
 
-const FmtErrEmptyMessage = `commit with no message without specifying the "--allow-empty-message" flag`
+const fmtErrEmptyMessage = `commit with no message without specifying the "--allow-empty-message" flag`
+const internalErrorCode = 2
 
 const (
-	MessageFlagName           = "message"
-	AllowEmptyMessageFlagName = "allow-empty-message"
-	MetaFlagName              = "meta"
-	CommitCreateTemplate      = `Commit for branch "{{.Branch.Ref}}" completed.
+	dateFlagName              = "epoch-time-seconds"
```

Comment:
I realize that the name is not reflecting the flag's purpose, but this is kind of what I was aiming for because of its "plumbingness" (use it only if you are 100% sure you understand what it does).
WDYT?